RecoLocalTracker/SiStripRecHitConverter/src/SiStripRecHitMatcher.cc

```diff
@@ -96,8 +96,8 @@ void SiStripRecHitMatcher::match(const SiStripRecHit2D* monoRH,
   // position of the initial and final point of the strip in local coordinates (mono det)
   StripPosition stripmono = StripPosition(topol.localPosition(RPHIpointini), topol.localPosition(RPHIpointend));
 
-  if (trackdirection.mag2() <
-      FLT_MIN) {  // in case of no track hypothesis assume a track from the origin through the center of the strip
+// in case of no track hypothesis assume a track from the origin through the center of the strip
```

Comment:
Shouldn't this comment be indented by two more spaces? Or did clang-format place it here?